.gitignore

```diff
@@ -4,3 +4,4 @@ logs/*
 node_modules/*
 .DS_Store
 .idea
+*.env
```

Comment:
In Milo we just have `.env` excluded, just FYI.

Reply:
That's good to know @overmyheadandbody; so should I delete this PR and instead just mark the Kodiak finding as not applicable, or are you suggesting I change the `*.env` to just `.env` ?